flink-libraries/flink-table/src/test/java/org/apache/flink/table/api/java/batch/TableEnvironmentTest.java

```diff
@@ -47,10 +48,16 @@
 import org.junit.runners.Parameterized;
 
 @RunWith(Parameterized.class)
-public class TableEnvironmentITCase extends TableProgramsCollectionTestBase {
+public class TableEnvironmentTest {
 
-	public TableEnvironmentITCase(TableConfigMode configMode) {
-		super(configMode);
+	private TableConfig config;
+	private BatchTableTestUtil util = new BatchTableTestUtil();
+
+	public TableEnvironmentTest(TableProgramsTestBase.TableConfigMode configMode) {
```

Comment:
This is not a TableProgramsTestBase anymore so we can get rid of this constructor and the parameters and simplify the class.